tests/base.py

```diff
@@ -11,6 +11,18 @@
 from sitewit.services import SitewitService
 
 
+def dictcontainsdict(d1, d2):
```

Comment:
but that's a helper for unittests, nope?  Can we at least separate words in the function name with `_`